### ISSUE TITLE
CW 2.0 Reload animations for wall weapons are broken on my server.

### ISSUE THREAD

**Issue by Coract** [Zet0rz/nzombies#527]
*using master-workshop, if i switch to dev branch it magically works... *both of my dedicated and local servers but not sp
Only weapons from the CW 2.0 pack or versions of it that are BOUGHT OFF THE WALL or put in through cheats don't have reload animations. Ones from the box do have working animations.
EXAMPLE VIDEO : https://youtu.be/VL_MPMbqIbk (no errors in console either)

**Comment by Coract** [comment]
I also tried Pack a Punching them but it didn't change it, neither did getting speed cola.

**Comment by Coract** [comment]
I installed gmod and NZombies on my brother's pc and it didn't work either so it can't just be me

**Comment by Coract** [comment]
Still no way for me to fix ::((

**Comment by DjNathann** [comment]
Its a creators of the weapon not the gamemode.

Close it.

**Comment by minin43** [comment]
No, this is not a problem with the creator of the weapons. I experience this bug all the time on all different nZ servers. It's a problem with how ammo is distributed and it has been modified by Zet0r. He modified it to avoid scenarios where you pick up guns with the exact same ammo type and they then share the ammo. If you notice in the 3D2D HUD, anytime the reload animation doesn't play, the gun doesn't have any CW2.0 ammo.

**Comment by Coract** [comment]
*The conversing continues

**Comment by YuRaNnNzZZ** [comment]
You have some addons that conflicts with CW2.0 or nZombies then.
Everything is working fine for me with nZombies (master-workshop branch) and CW2.0 installed.
https://youtu.be/LZKD7-mTWCk

**Comment by Coract** [comment]
Is that Multiplayer? (also those were the only addons I had at the time.)

**Comment by YuRaNnNzZZ** [comment]
No, it was singleplayer. Just tried it with listen server and yes, CW2.0 guns bought from walls don't reload.
Must be a problem with synchronizing ammotypes or something.

**Comment by minin43** [comment]
So this is officially a bug that *needs to be looked into,* let's get it on the "to-do" list and not on the "closed" list.

I hate this bug the most.

**Comment by DjNathann** [comment]
Just tried some CW 2.0 Weapons and yea some of them doesn't do the reload animation but still reloads.

**Comment by Coract** [comment]
How can I make this problem relevant again?

**Comment by minin43** [comment]
Just leave the thread as it is. It will be addressed in due time and Zet0r will close the problem himself when it's tested and fixed.

**Comment by Coract** [comment]
Okay
